src/pybind/mgr/dashboard/controllers/cephfs.py

```diff
@@ -384,8 +382,8 @@ def _set_ls_dir_path(self, path):
             path = os.path.normpath(path)
         return path
 
-    @RESTController.Resource('POST')
-    def mk_dirs(self, fs_id, path):
+    @RESTController.Resource('POST', path='/tree')
```

Comment:
It's used in our API tests. They are not yet used in the UI as it's not sure yet if it would be very useful as one should mount the cephfs to work with it by creating directories and files.